tests/visualization/test_open3d_vis_utils.py

```diff
@@ -0,0 +1,35 @@
+"""
+Unit tests for Open3d visualizaation code.
+
+Author: John Lambert
+"""
+
+import numpy as np
+import open3d
+
+import visualization.open3d_vis_utils as open3d_vis_utils
+
+
+def test_create_colored_spheres_open3d() -> None:
+    """Try creating spheres centered at randomly generated 3d locations (disclaimer: slow)."""
+    num_points = 1000
+    point_cloud = np.random.randn(num_points, 3)
```

Comment:
we should set the seed before using random in tests for reproducibility.

Reply:
I added a random seed. Although these viz tests should work for any amount of randomness : - )